scripts/run_e2e_tests.py

```diff
@@ -164,6 +163,48 @@
 # the oppia web server.
 SUBPROCESSES = []
 
+RERUN_POLICIES = {
+    'accessibility': 'never',
```

Comment:
Please add a code comment explain the definition of never, always and known flakes and how it affects the rerun.

Reply:
And define these values in constant and use it here

Reply:
done